README.md

```diff
@@ -1,3 +1,9 @@
 # Using Endpoints on Google Compute Engine or Container Engine
 
 This repository contains samples and utilities for using Google Cloud Endpoints v2 on Compute Engine or Container Engine
+
+Folder structures:
+* gettting-started: ESPv2 GKE depolyment yaml files used Cloud Endpoint [tutorials](https://cloud.google.com/endpoints/docs/openapi/tutorials).
+* gke: ESPv2 deployment yaml files for GKE.
```

Comment:
Nit: GKE -> Google Kubernetes Engine

No need to change folder name, just descriptions across README